meta-webserver/recipes-webadmin/cockpit/cockpit_218.bb

```diff
@@ -18,6 +18,12 @@ inherit gettext pkgconfig autotools systemd distro_features_check
 DEPENDS += "glib-2.0-native intltool-native gnutls"
 DEPENDS += "virtual/gettext json-glib krb5 libpam"
 
+python __anonymous () {
+    # only works on glibc systems
+    if d.getVar('TCLIBC') != "glibc":
+        raise bb.parse.SkipRecipe("incompatible with %s C library" % d.getVar('TCLIBC'))
+}
```

Comment:
Just add 
```
COMPATIBLE_HOST_libc-musl = "null"
```

Reply:
Great!
Now I have combined all inputs to one commit. Hope everything fits :-)